doc/tutorials/getting_started/getting_started.rst

```diff
@@ -49,24 +45,16 @@ Move into your Colcon workspace and pull the MoveIt tutorials source: ::
   cd ~/ws_moveit2/src
   git clone https://github.com/ros-planning/moveit2_tutorials -b humble --depth 1
 
-Next we will download the source code for the rest of MoveIt: ::
-
-  vcs import < moveit2_tutorials/moveit2_tutorials.repos
-
-The import command may ask for your GitHub credentials. You can just press Enter until it moves on (ignore the "Authentication failed" error).
-
 Build your Colcon Workspace
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 The following will install from Debian any package dependencies not already in your workspace. This is the step that will install MoveIt and all of its dependencies: ::
 
   sudo apt update && rosdep install -r --from-paths . --ignore-src --rosdistro $ROS_DISTRO -y
```

Comment:
If you just follow the tutorials blindly this var never gets set. Should make a note to add it to `~/.bashrc` or something, or since we're using humble everywhere else:

```suggestion
  sudo apt update && rosdep install -r --from-paths . --ignore-src --rosdistro humble -y
```

Reply:
you are reading the source, it get's replace in the rendered version.